mage/algorithms/machine-learning-graph-analytics/temporal-graph-networks.md

```diff
@@ -0,0 +1,69 @@
+---
+id: temporal-graph-networks
+title: Temporal Graph Networks
+sidebar_label: Temporal Graph Networks
+---
+
+## Description
+
+The **temporal_graph_networks (TGNs)** is a **[graph neural network (GNN)](https://distill.pub/2021/gnn-intro/)** method on dynamic graphs. 
```

Comment:
This sentence could be unclear since “method” has a specialized use in software. Suggest rewriting it as “TGNs are a type of GNNs for dynamic graphs”.

Reply:
Agree with this comment